controls/radpdfviewer/features/find.md

```diff
@@ -0,0 +1,147 @@
+        
+    Dim searchResult As SearchResult = Me.pdfViewer.Find(sampleText, textSearchOptions)
+{{endregion}}
+
+The **FindAll** method has two overloads:
+* **FindAll(string text)**: Finds all matches for the specified text in the current document.
+* **FindAll(string text, TextSearchOptions options)**: Finds all matches for the specified text in the current document using specified options.
+
+#### __[C#] Example 2: Find all matches for the specified text using specified options__
+
+{{region radpdfviewer-find_1}}
+
+    IEnumerable<SearchResult> searchResult = this.pdfViewer.FindAll(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 2: Find all matches for the specified text using specified options__
+
+{{region radpdfviewer-find_1}}
+
+    Dim searchResult As IEnumerable(Of SearchResult) = Me.pdfViewer.FindAll(sampleText, textSearchOptions)
+{{endregion}}
+
+The **FindPrevious** method has two overloads:
+* **FindPrevious(string text)**: Finds the previous text in the current document.
+* **FindPrevious(string text, TextSearchOptions options)**: Finds the previous text in the current document using specified options.
+
+#### __[C#] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    SearchResult searchResult = this.pdfViewer.FindPrevious(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    Dim searchResult As SearchResult = Me.pdfViewer.FindPrevious(sampleText, textSearchOptions)
+{{endregion}}
+
+## SearchResult
+
+The **SearchResult** class holds the result from a specific search command. This class exposes several properties and methods:
+* **NotFound**: A static property which gets a default SearchResult instance when there is no search result found.
+* **Range**: Gets the result text range position.
+* **Result**: Gets the result as text.
+* **ToString**: Returns a string that represents the current object.
+
+#### __[C#] Example 4: Check if a match was found__
+
+{{region radpdfviewer-find_3}}
+
+    if (searchResult != SearchResult.NotFound)
+    {
+        TextRange searchResultRange = searchResult.Range;
+        string searchResultAsText = searchResult.Result;
+    }
+{{endregion}}
+
+#### __[VB.NET] Example 4: Check if a match was found__
+
+{{region radpdfviewer-find_3}}
+
+    If searchResult <> SearchResult.NotFound Then
+        Dim searchResultRange As TextRange = searchResult.Range
+        Dim searchResultAsText As String = searchResult.Result
+    End If
+{{endregion}}
+
+## TextSearchOptions
+
+The **TextSearchOptions** class holds all possible search options when invoking finding. The class has various constructors that can be used depending on the necessities. Their main difference is they are receiving different amount of parameters according to the selected options.
```

Comment:
The **TextSearchOptions** class holds all possible search options when invoking one of the Find methods.